tests/virtualization/universal/waitfor_guests.pm

```diff
@@ -16,6 +16,18 @@ use testapi;
 use utils;
 use version_utils 'is_sle';
 
+# Open vnc and take screenshot of guests
+sub guests_vnc_screenshot {
+    foreach my $guest (keys %virt_autotest::common::guests) {
+        # Wait for virt-viewer to fully connect and display VNC stream
+        enter_cmd "virt-viewer -f $guest & sleep 21 && killall virt-viewer";
+        sleep 19;
+        record_info "$guest", "$guest screenshot";
+        save_screenshot();
+        sleep 6;
```

Comment:
@tbaev  Hi Teodor, thanks for the PR. Finally, we can have more info about where the guest installation fails! This is a light enough PR, and this behavior of taking screenshots overall takes ~4 min which is not long.   Good job!

I only have some minor comments.
1) about function name `guests_vnc_screenshot`, generally it is suggested to start function name with a VERB
2) would you please explain a little bit about how the time 21/19/6 are chosen? I guess '19 seconds' is chosen  to let virt-viewer open the window (why so long)? 21-19 (so 2) seconds are left to take the screenshot (is it a little short)? the last sleep 6, what are the reasons?

Reply:
1. I can change it to screenshot_vnc_guests?
2. Virt-viewer usually will open inside openQA and start rendering VNC faster then 19 seconds, but sometimes on some products / hardware (for example sles12-sp5) it will take almost 19 seconds (this delay is only when openQA is starting virt-viewer). OpenQA is fast at taking the screenshot, I have found 2 seconds is always working.

Virt-viewer is open for 21 seconds after that the kill command is send, we are waiting additional 4 seconds (25 seconds total) to be sure that virt-viewer is fully closed before running the function for the next guest.

Reply:
Yes, I have. My first implementation was with needles but after discussion in the progress ticket and finding some issues I decided it was best to use 'sleep' in this scenario.

Reply:
OK, cool!